specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -164,9 +164,11 @@ export function InteractionDialog({
       return;
     }
     const catalogNumbers = prepsData.map(([catalogNumber]) => catalogNumber);
-    const missing = catalogNumbers.filter(
-      (catalogNumber) => !entries?.includes(catalogNumber)
-    );
+    const missing = entries
```

Comment:
please please don't do implicit conversion to boolean
do explicit `typeof entries === 'object'` instead to avoid bugs

Reply:
given that this is high urgency pull request, @CarolineDenis please open a new pull request with the fix for this lower priority issue

please also keep the implicit vs explicit boolean comparison in mind in the future